dockers/docker-fpm-frr/frr/bgpd/bgpd.main.conf.j2

```diff
@@ -41,6 +41,7 @@ router bgp {{ DEVICE_METADATA['localhost']['bgp_asn'] }}
 {% block bgp_init %}
   bgp log-neighbor-changes
   no bgp default ipv4-unicast
+  no bgp ebgp-requires-policy
```

Comment:
what is this policy?

Reply:
http://docs.frrouting.org/en/latest/bgp.html?highlight=ebgp-policy#require-policy-on-ebgp
incoming and outcoming filters